models/prices/prices_usd_daily.sql

```diff
@@ -0,0 +1,24 @@
+{{ config(
+        schema='prices',
+        alias = 'usd_daily',
+        materialized = 'table',
+        file_format = 'delta',
+        post_hook = '{{ expose_spells(\'["ethereum", "solana", "arbitrum", "base", "gnosis", "optimism", "bnb", "avalanche_c", "polygon", "scroll", "zksync"]\',
+                                    "sector",
+                                    "prices",
+                                    \'["aalan3"]\') }}'
+        )
+}}
+
+select * from (
+select
+    cast(date_trunc('day', minute) as date) as day,
+    blockchain,
+    contract_address,
+    decimals,
+    symbol,
+    price,
```

Comment:
After discussing with @0xRobin we should use avg here instead

Reply:
update: added all of OHLC (open-high-low-close) price points to the schema